samples/matter/weather_station/README.rst

```diff
@@ -100,7 +123,8 @@ Commissioning the device
     :end-before: matter_door_lock_sample_commissioning_end
 
 To start the commissioning procedure, the controller must get the commissioning information from the Matter accessory device.
-The data payload, which includes the device discriminator and setup PIN code, is encoded within a QR code, available through the RTT interface, and can be shared using an NFC tag.
+The data payload, which includes the device discriminator and setup PIN code, is encoded within a QR code and shared using a NFC tag.
```

Comment:
Maybe skipping the "QR code" part would be a good idea? We know that it means that the NFC uses the same payload as the QR code, but for users it may be confusing - "what does the QR code have to do with NFC"?